release/build_pip_package.sh

```diff
@@ -16,44 +16,58 @@
 set -e
 set -x
 
+# Pick the Python that TFQ/TensorFlow used during configure/build.
+# Order: explicit env -> 3.11 -> python3
+PY="${PYTHON_BIN_PATH:-}"
+if [[ -z "$PY" ]]; then
+  if command -v python3.11 >/dev/null 2>&1; then
+    PY="$(command -v python3.11)"
+  elif command -v python3 >/dev/null 2>&1; then
+    PY="$(command -v python3)"
+  else
+    echo "ERROR: No suitable python found. Set PYTHON_BIN_PATH." >&2
+    exit 2
+  fi
+fi
+echo "Using Python: $PY"
+
+# Ensure packaging tools are present in THIS interpreter
+"$PY" - <<'PY' || { "$PY" -m pip install --upgrade pip setuptools wheel; }
+import importlib
+for m in ["setuptools","wheel"]:
+    importlib.import_module(m)
+PY
+
 EXPORT_DIR="bazel-bin/release/build_pip_package.runfiles/__main__"
 
-function main() {
-  DEST=${1}
-  EXTRA_FLAGS=${2}
+main() {
+  DEST="$1"
+  EXTRA_FLAGS="$2"
 
-  if [[ -z ${DEST} ]]; then
+  if [[ -z "$DEST" ]]; then
```

Comment:
Here and elsewhere that variables are referenced, could we keep the recommended Google style for shell scripts? In particular, for this case the section on [variable expansion](https://google.github.io/styleguide/shellguide.html#variable-expansion) is relevant, and the preference is for brace-delimiting variables: "${DEST}".